debug/heatmap-layer.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+    <title>Create a heatmap layer</title>
+    <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no">
+    <link rel='stylesheet' href='../dist/mapbox-gl.css' />
+    <style>
+        body { margin: 0; padding: 0; }
+        #map { position: absolute; top: 0; bottom: 0; width: 100%; }
+        #globe { 
+            position: absolute;
+            top: 0; left: 0;
+            margin: 15px;
+            border: none;
+            font-size: 14px;
+            padding: 8px;
+            background-color: white;
+        }
+    </style>
+    </style>
+</head>
+
+<body>
+<div id='map'></div>
+<button id='globe'>switch to globe</button>
+
+<script src='../dist/mapbox-gl-dev.js'></script>
+<script src='../debug/access_token_generated.js'></script>
+<script>
+
+const map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/dark-v10',
+    center: [-120, 50],
+    zoom: 2
+});
+
+document.getElementById('globe').addEventListener('click', (e) => {
+    if (e.target.innerHTML === 'switch to globe') {
```

Comment:
More of a usability issue, but I'd either change the color of the button, or add an atmosphere to this page, it's hard to see that it's a button with white on white otherwise:

![Screenshot from 2022-05-09 12-12-24](https://user-images.githubusercontent.com/7061573/167480930-6fe474c9-a375-416b-8550-bb6047cd2eb0.png)

Reply:
Ok will change!

Reply:
Updated to be more visible - I'd like to keep the focus on heatmap testing example without atmosphere to not add too many elements that someone might think they are testing for on the page (adding separate PR for atmosphere and globe test shortly!)
<img width="162" alt="Screen Shot 2022-05-09 at 12 27 08 PM" src="https://user-images.githubusercontent.com/42715836/167483224-22012df8-795f-443c-bdc9-758e4e919567.png">